standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -239,6 +239,10 @@ private void startLocalSegmentStores() throws Exception {
      * @param segmentStoreId id of the SegmentStore.
      */
     private void startLocalSegmentStore(int segmentStoreId) throws Exception {
+        System.setProperty("pravega.client.auth.method", "Default");
```

Comment:
You can create a new Properties() object and add it to the `configBuilder` using `include`. Most likely right after `System.getProperties` are included.

Reply:
Fixed.